sdk/search/Microsoft.Azure.Search/tests/Microsoft.Azure.Search.Tests.csproj

```diff
@@ -20,6 +20,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="System.Text.Encodings.Web" VersionOverride="4.5.1" />
```

Comment:
Same question as above.

Reply:
@heaths Why the change to a Track 1 project?

Reply:
The version that was referenced automatically was 4.5.0, and guidance is to use 4.5.1 for 4.0.0 - 4.5.0. 4.7.2 will update many more references than we want.

As for the track 1, it's because our repo is getting flagged with a CVE because of a vulnerable System.Text.Encodings.Web. It's merely a test and not harmful. Better to fix it than seek an exemption that goes way up the chain.